lib/config/httpRoutes.js

```diff
@@ -209,47 +208,45 @@ module.exports = [
 
 
   // DELETE
-  {verb: 'delete', url: '/:index/:collection/_truncate', controller: 'collection', action: 'truncate'},
   {verb: 'delete', url: '/:index/:collection/_specifications', controller: 'collection', action: 'deleteSpecifications'},
+  {verb: 'delete', url: '/:index/:collection/_truncate', controller: 'collection', action: 'truncate'},
 
-  {verb: 'delete', url: '/:index/:collection/_query', controller: 'document', action: 'deleteByQuery'},
   {verb: 'delete', url: '/:index/:collection/:_id', controller: 'document', action: 'delete'},
+  {verb: 'delete', url: '/:index/:collection/_query', controller: 'document', action: 'deleteByQuery'},
 
-  {verb: 'delete', url: '/_mdelete', controller: 'index', action: 'mDelete'},
   {verb: 'delete', url: '/:index', controller: 'index', action: 'delete'},
+  {verb: 'delete', url: '/_mdelete', controller: 'index', action: 'mDelete'},
 
-  {verb: 'delete', url: '/roles/:_id', controller: 'security', action: 'deleteRole'},
   {verb: 'delete', url: '/profiles/:_id', controller: 'security', action: 'deleteProfile'},
+  {verb: 'delete', url: '/roles/:_id', controller: 'security', action: 'deleteRole'},
   {verb: 'delete', url: '/users/:_id', controller: 'security', action: 'deleteUser'},
 
+  {verb: 'delete', url: '/ms/:_id', controller: 'ms', action: 'del'},
+  {verb: 'delete', url: '/ms', controller: 'ms', action: 'del'},
```

Comment:
the `del` can delete multiple elements at a time. _id can be provided but also the argument `keys`. So the user can use a route like DELETE /ms/?keys[]=foo&keys[]=bar